### DISCUSSION TITLE
Upvote Features You Want! 给你最想要的功能投票吧！

### DISCUSSION THREAD

**Discussion by wzhudev** [dream-num/univer#1024] [Ideas | 产品建议]
Hi everyone. Here is a poll for you to pick the features you want! Import & export, printing and find & replace is under development, so they are not in the voting options.

Links:

* [Facade API](https://univer.ai/guides/facade/)

---

大家好。为了了解大家的需求，我们设置了这个投票，你可以选择一个你目前最想要的功能！导入导出、打印和查找替换已经在开发中，所以不在本次投票范围内。

如果想要的功能没有出现在选项中，欢迎在下方留言。注意留言前搜索一下是不是已经有其他人提出同样的需求了，那么你只需要点亮 👍 即可。

链接：

* [Facade API](https://univer.ai/guides/facade/)

**Comment by Delan1026** [comment]
univer现在如何导出啊？在我的使用场景中需要把表格做个快照引用到其他文档中，所以希望导出图片

**Comment by 352836670** [comment]
表格锁定、单元格锁定什么时候能释放出来啊

**Reply by yooyea** [reply]
https://github.com/dream-num/univer/discussions/1771

**Comment by fantasy-zeng** [comment]
什么时候支持浮动图片，粘贴图片呢

**Comment by jikkai** [comment]
> 什么时候支持浮动图片，粘贴图片呢

https://univer.ai/guides/sheet/roadmap

**Comment by semmywong** [comment]
I wish there was a feature to insert a background image

**Comment by Matt-Jensen** [comment]
More Formulas #908

**Comment by zhang-ke-ke** [comment]
希望老师把导入导出从后端服务中剥离出来，这两个应该是基础功能，大部分场景应该都需要，但是去部署一套后端太不值当了

**Reply by jikkai** [reply]
We do not have this plan, the reason has been explained in the document.

**Reply by zhang-ke-ke** [reply]
等待万人血书，加入这个计划，这个对前端太不友好了

**Reply by lovert9999** [reply]
如果需要前端导入需要自己写插件，这个官方文档API依据给了IWorkbookData，IWorksheetData， ICellData的接口了，只需要自己通过其他的EXCEL解析库将excel文件转成IWorkbookData，自行导入
